eng/emsdk_env.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
```

Comment:
I don't think we need the shebang since this file is intended to be sourced.

Reply:
We don't need it, but it should avoid weird behaviour if the file is _not_ sourced.